R/check.R

```diff
@@ -5,6 +5,8 @@
 #' elsewhere in your function, make sure to use `add = TRUE` so that you
 #' don't override the handler set up by `check_dots_used()`.
 #'
+#' @param action The action to take when the dots have not been used. One of
```

Comment:
An alternative interface would be to have separate functions for all of these examples, but 3 (4?) types of signal + 3 different check functions is at least 9-12 functions, which seemed like a lot to me, which is why I went the argument route.

Reply:
If anyone has alternative names to suggest I would be happy to change this, I first thought of `type`, but thought `action` captured the intent better.